lib/helpers/utils.js

```diff
@@ -762,6 +762,8 @@ export function getAllFiles(dirPath, pattern, options = {}) {
   // Only ignore node_modules if the caller is not looking for package.json
   if (!pattern.includes("package.json") && !options.includeNodeModulesDir) {
     ignoreList.push("**/node_modules/**");
+  } else if (options.includeNodeModulesDir === false) {
```

Comment:
Ignoring the node_modules improves performance when the goal is to only collect package.json files to understand the workspace dependencies.